Build/Common.Build.props

```diff
@@ -86,6 +86,8 @@
         %(PreprocessorDefinitions);
         BYTECODE_TESTING=1
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(BuildWabt)'=='true'">%(PreprocessorDefinitions);EXCLUDE_WABT=0</PreprocessorDefinitions>
```

Comment:
Right now we already use `ENABLE_WABT` in the code. `EXCLUDE_WABT` is only use to not set `ENABLE_WABT` along with some other conditions in CommonDefines.h